packages/@aws-cdk/cdk/lib/pseudo.ts

```diff
@@ -1,7 +1,16 @@
+import { CfnReference } from './cfn-reference';
 import { Construct } from './construct';
-import { Reference } from './reference';
 import { Token } from './token';
 
+const AWS_ACCOUNTID = 'AWS::AccountId';
```

Comment:
Why are these consts needed? Aren't they used exactly once?

Reply:
Once for the scoped, once for the unscoped variant.